server-spi/src/main/java/org/keycloak/policy/PasswordPolicyProvider.java

```diff
@@ -41,4 +41,11 @@ default Integer parseInteger(String value, Integer defaultValue) {
         }
     }
 
+    default Double parseDouble(String value, Double defaultValue) {
+        try {
+            return value != null ? Double.valueOf(value) : defaultValue;
+        } catch (NumberFormatException e) {
+            throw new PasswordPolicyConfigException("Not a valid number");
+        }
+    }
```

Comment:
I think that this part is not needed if a spi config is used. WDYT?

Reply:
Thanks for the hint! Yes that was an oversight, I just removed the code.